src/Service/StaleIssueCommentGenerator.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Service;
+
+use App\Api\Issue\IssueType;
+
+/**
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+class StaleIssueCommentGenerator
+{
+    /**
+     * Get a comment to say: "I'm closing this now".
+     */
+    public function getClosingComment(): string
+    {
+        return <<<TXT
+Hey,
+
+I didn't hear anything so I'm going to close it. Feel free to comment if this is still relevant, I can always reopen!
+TXT;
+    }
+
+    /**
+     * Get a comment that encurage users to reply or close the issue themselves.
+     *
+     * @param string $type Valid types are IssueType::*
+     */
+    public function getComment(string $type): string
+    {
+        switch ($type) {
+            case IssueType::BUG:
+                return $this->bug();
+            case IssueType::FEATURE:
+            case IssueType::RFC:
+                return $this->feature();
+            default:
+                return $this->unknown();
+        }
+    }
+
+    private function bug(): string
+    {
+        return <<<TXT
+Hey,
+Is this bug still relevant? Have you managed to find a workaround?
+TXT;
+    }
+
+    private function feature(): string
+    {
+        return <<<TXT
+Thank you for this suggestion.
+Is this still something we want to do?
```

Comment:
I think the "we want to do" can be understood as "the core team needs to say YES" instead of "I need to engage with this issue". Maybe something like "Would you still like to see this feature?" or the like?